src/serac/physics/utilities/functional/detail/hexahedron_L2.inl

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2019-2021, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/**
+ * @file hexahedron_L2.inl
+ *
+ * @brief Specialization of finite_element for L2 on hexahedron geometry
+ */
+
+// interpolate at Gauss-Lobatto nodes for the appropriate polynomial order

```

Comment:
Also i think this should be Gauss-Legendre...

Reply:
```suggestion
// interpolate at Gauss-Legendre nodes for the appropriate polynomial order
```